src/core/PAGE_main.jl

```diff
@@ -339,6 +339,9 @@ function perturb_marginal_page_emissions!(base::Model, marginal::Model, gas::Sym
         update_param!(marginal.mi.md, :add, forcing_shock)
     end
 
+    # create a new model from the marginal model's model instance
+    marginal = Model(marginal.mi)
```

Comment:
@ckingdon95 this (potentially in a crude or slow way) fixes the `compute_ssc(PAGE, ...)` call, which was broken by Mimi v1.2.0.  It has something to do with model refs, instances, and the instance of a def, because _without_ this line, the code
```
    if year != nothing
        run(base)
        Mimi.build!(marginal)
        marginal = perturb_marginal_page_emissions!(base, marginal, gas, year)
        run(marginal)
    end
``` 
doesn't have any effect i.e. `perturb_marginal_page_emissions!` doesn't stick through the `run` phase